### DISCUSSION TITLE
Progress bar for waiting for the long pending background task

### DISCUSSION THREAD

**Discussion by mayankjoshii** [hecrj/iced#1824] [General]
I am running a long pending background job and I want to show a progress bar until the job is not completed, but the challenge is that I don't know when the job will finish. So I want to show a progress bar that goes from left to right and right to left until the job is not complete. 

Here's the example of progress bar I want to implement. 

![image](https://user-images.githubusercontent.com/98211866/235924799-50babfc6-a43e-4f8a-aaa5-ddba22c752c6.png)

I'm looking for the ways to implement this, and I want to know if there's any property (ex: progressbar isindeterminate= false) so that I can implement such progress bar.

**Comment by 13r0ck** [comment]
You will need a custom widget. Look at the spinner widgets in https://github.com/iced-rs/iced_aw
